setup.py

```diff
@@ -29,11 +29,11 @@
           'pandas>=0.23.0',
           'GitPython>=2.1.3',
           'xlsxwriter>=1.0.2',
-          'numpy>=1.12.1'
+          'numpy>=1.12.1',
+          'green>=2.13.0'
       ],
-      test_suite='nose.collector',
-      tests_require=['nose'],
       packages=find_packages(),
       include_package_data=True,
-      scripts=['bin/staramr']
+      scripts=['bin/staramr'],
+      setup_requires = ['green']
```

Comment:
Could you test out to see if only only need to set `green` in the **test_requires** parameter instead?  I am not sure as to exactly what the differences is between **setup_requires** and **test_requires**, but I would imagine since `green` is only for testing, it should be set there.

If this doesn't work, keeping it in **setup_requires** is okay.

Reply:
Resolved this in 5c946ea, turns out having the `tests_require` parameter worked well